tests/unit/v2/API/internal/testPrometheusMetrics.js

```diff
@@ -0,0 +1,35 @@
+const assert = require('assert');
+const { RequestContext } = require('../../../../../libV2/models');
+const { templateRequest } = require('../../../../utils/v2Data');
+const prometheusMetrics = require('../../../../../libV2/server/API/internal/prometheusMetrics');
+
+describe('Test prometheusMetrics', () => {
+    const overrides = {
+        swagger: {
+            operation: {
+                'x-router-controller': 'internal',
+                'operationId': 'prometheusMetrics',
+            },
+            params: {},
+        },
+    };
+    const ctx = new RequestContext(templateRequest(overrides));
+
+    before(async () => {
+        await prometheusMetrics(ctx);
+    });
+
+    it('should set statusCode to 200', () => {
+        assert(ctx.results.statusCode === 200);
```

Comment:
We usually use `assert.strictEqual(a, b)` for all such test cases, it can give a better output when it fails